test/testdata/lsp/show_symbol.rb

```diff
@@ -0,0 +1,58 @@
+# typed: strict
+
+module First
```

Comment:
Can you do a packaging test too? Stripe has packaging mode on which has a material impact on this feature.